Model/Sales/Repository/PackageRepository.php

```diff
@@ -23,15 +23,20 @@
 
 class PackageRepository extends Package
 {
-    public const DEFAULT_MAILBOX_WEIGHT       = 2000;
-    public const DEFAULT_DIGITAL_STAMP_WEIGHT = 2000;
-    public const DEFAULT_LARGE_FORMAT_WEIGHT  = 2300;
+    public const MAXIMUM_MAILBOX_WEIGHT       = 2000;
+    public const MAXIMUM_DIGITAL_STAMP_WEIGHT = 2000;
+    public const MAXIMUM_LARGE_FORMAT_WEIGHT  = 2300;
```

Comment:
2300 is ook niet veel voor large format...

Reply:
done